aws/app/lambda/reliability/reliability.js

```diff
@@ -24,7 +24,7 @@ exports.handler = async function (event) {
       securityAttribute: messageData.Item?.SecurityAttribute.S ?? "Unclassified",
     }
 
-    const {submissionID, formSubmission, formID, sendReceipt, createdAt, language} = processedMessageData
+    const {submissionID, formSubmission, formID, sendReceipt, createdAt, language, securityAttribute} = processedMessageData
```

Comment:
In the getSubmission function the securityAttribute property is not returned.  The DynamboDB ProjectExpression does not include that field: ProjectExpression: "SubmissionID,FormID,SendReceipt,FormData,FormSubmissionLanguage,CreatedAt"